lua/telescope/entry_manager.lua

```diff
@@ -65,18 +78,28 @@ function EntryManager:get_container(index)
   return {}
 end
 
+--- Get the entry in the EntryManager corresponding to `index`
+---@param index number
+---@return table: table with information about the given entry
 function EntryManager:get_entry(index)
   return self:get_container(index)[1]
 end
 
+--- Get the score in the EntryManager corresponding to `index`
+---@param index number
+---@return number
 function EntryManager:get_score(index)
   return self:get_container(index)[2]
 end
 
+--TODO: documentation
```

Comment:
Ordinal is the text that is filtered on. So you can have a different display than what you filter on. Right now tj isn't doing anything with `get_ordinal` tho